DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -211,7 +211,7 @@ public void RegisterCentreManager(AdminRegistrationModel registrationModel, int
 
             CreateDelegateAccountForAdmin(registrationModel, jobGroupId);
 
-            registrationDataService.RegisterAdmin(registrationModel);
+            registrationDataService.RegisterAdmin(registrationModel, 0);

```

Comment:
Probably need to stick a TODO against both of these 0s in this file just to make absolutely sure they get done.